mars/services/task/execution/ray/executor.py

```diff
@@ -70,6 +69,8 @@ def execute_subtask(
     input_keys: List[str],
     *inputs,
 ):
+    from .....optimization.physical import optimize
```

Comment:
Import inside the function makes it runs slow. Also it's better to import at top of the file: https://peps.python.org/pep-0008/#imports

Reply:
When sticking to fast import we have no choices but make compromises. @chaokunyang any ideas?